dev-vcs/git-repair/git-repair-1.20220404.ebuild

```
@@ -0,0 +1,47 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.0.0
+
+CABAL_FEATURES="haddock hoogle profile"
+inherit haskell-cabal
+
+DESCRIPTION="repairs a damaged git repository"
+HOMEPAGE="https://git-repair.branchable.com/"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+GHC_BOOTSTRAP_PACKAGES=(
+	async
+	filepath-bytestring
+	IfElse
+	split
+	time
+	unix-compat
+)
```

It looks like you missed a couple, at least based on `Setup-Depends` (under `custom-setup`) in [the `.cabal` file](https://hackage.haskell.org/package/git-repair-1.20220404/git-repair.cabal).

Also, `time` isn't needed here (although it wouldn't hurt) since it's [already exposed by default](https://github.com/gentoo-haskell/gentoo-haskell/blob/ad4015edfc83519a639ddef4fa542a109798d888/eclass/haskell-cabal.eclass#L330-L355).

```suggestion
GHC_BOOTSTRAP_PACKAGES=(
	async
	data-default
	filepath-bytestring
	hslogger
	IfElse
	split
	unix-compat
)
```